### PR TITLE
fix: now properly handles legacy libp2p-webrtc-multiaddrs

### PR DESCRIPTION
**Pull request by diasdavid**
//cc @dignifiedquire

### CODE REVIEW

**Comment by diasdavid** [comment]
one sec, there is more :)

**Comment by diasdavid** [comment]
Ok, now yes :)

File: src/index.js [hunk 1]
```diff
@@ -218,8 +218,18 @@ class WebRTCStar {
...
+      maStr = maStr.substring(legacy.length, maStr.length)
+      let ma = multiaddr(maStr)
+      const tuppleIPFS = ma.stringTuples().filter((tupple) => {
+        return tupple[0] === 421 // ipfs code
+      })[0]
+
+      ma = ma.decapsulate('ipfs')
+      ma = ma.encapsulate('p2p-webrtc-star')
+      ma = ma.encapsulate(`/ipfs/${tuppleIPFS[1]}`)
+      maStr = ma.toString()
```
Comment (dignifiedquire) [inline src/index.js]: how about you pull this into it's own method and write a test for it ? :)

Comment (diasdavid) [reply src/index.js]: sounds good :)

**Comment by diasdavid** [comment]
added test

**Pull request merged**